source/fundamentals/connection/connect.txt

```diff
@@ -77,12 +77,36 @@ In the example, we set two connection options: ``maxPoolSize=20`` and
 
 .. _connect-atlas-java-driver:
 
-The following code shows how you can use the sample connection URI in a client
-to connect to MongoDB.
+Atlas Connection Example
+------------------------
+
+To connect to a MongoDB deployment on Atlas, create a client. You can
+create a client that uses your connection string and other 
+client options by passing a ``MongoClientSettings`` object to the
+``MongoClients.create()`` method.
+
+To create a ``MongoClientSettings`` object, use the
+``MongoClientSettings.builder()`` method.
+
+To specify your connection URI, pass it to
+the ``ConnectionString`` constructor, pass the constructor to the
+``applyConnectionString()`` method, and chain this method to the
+``MongoClientSettings.builder()`` method. To specify any other client
```

Comment:
Issue:
I think there's too much information to follow in this sentence.

Suggestion:
Explain the purpose of the MongoClientSettings.builder() / MongoClientSettings before talking about applyConnectionString().

E.g.
"You can chain methods to the MongoClientSettings instance to specify your connection string and other connection options."

Once that information is established, it's probably simpler to describe how to include the connection URI. Also, in this instance, description of nesting doesn't seem necessary.

E.g.
"Call the applyConnection() method on your MongoClientSettings instance to specify your connection URI. You must pass the connection URI string to the ConnectionString() constructor since applyConnection() only accepts that type."